### PR TITLE
Added Appveyor build status button to the README file

### LINKED ISSUE #122: appveyor button missing
A button linking to the following link should be provided in README, as we now have appveyor builds

https://ci.appveyor.com/project/KodrAus/uuid

### PR DESCRIPTION
**Pull request by gowthamsadasivam**
This fixes #122 
Added Appveyor build status button to the README file and linked the button to https://ci.appveyor.com/project/KodrAus/uuid

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 uuid
 ====
 
-[![Build Status](https://travis-ci.org/uuid-rs/uuid.svg?branch=master)](https://travis-ci.org/uuid-rs/uuid) [![Latest Version](https://img.shields.io/crates/v/uuid.svg)](https://crates.io/crates/uuid) [![Join the chat at https://gitter.im/uuid-rs/Lobby](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/uuid-rs/Lobby?utm_source=badge&utm_medium=badge&utm_content=badge)
+[![Build Status](https://travis-ci.org/uuid-rs/uuid.svg?branch=master)](https://travis-ci.org/uuid-rs/uuid) [![Latest Version](https://img.shields.io/crates/v/uuid.svg)](https://crates.io/crates/uuid) [![Join the chat at https://gitter.im/uuid-rs/Lobby](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/uuid-rs/Lobby?utm_source=badge&utm_medium=badge&utm_content=badge) [![Appveyor Status](https://ci.appveyor.com/api/projects/status/github/uuid-rs/uuid?branch=master&svg=true)](https://ci.appveyor.com/project/KodrAus/uuid)
```
Comment (kinggoesgaming) [inline README.md]: Can you put the button next to travis button instead of at the end?

**Comment by KodrAus** [comment]
Thanks @gowthamsadasivam! Just a small nitpick: what do you think about moving the badge next to the Travis CI one, so the build badges are both together?

**Comment by kinggoesgaming** [comment]
omg i forgot to press submit on my review and Kodr ninja'd me

**Comment by gowthamsadasivam** [comment]
Hello @KodrAus & @kinggoesgaming 
As you've mentioned, I just moved the Appveyor button next to Travis CI one. 

Thanks.

**Pull request merged**